src/assets/images/tt_icon.svg

```diff
@@ -0,0 +1,7 @@
+<svg id="svg"
+	xmlns="http://www.w3.org/2000/svg"
+	xmlns:xlink="http://www.w3.org/1999/xlink" width="400" height="400" viewBox="0, 0, 400,400">
+	<g id="svgg">
+		<path id="path0" d="M21.917 171.587 C 22.001 360.472,21.785 341.343,23.935 349.739 C 29.871 372.912,48.700 391.472,72.030 397.146 C 79.484 398.959,78.758 398.923,109.889 399.038 L 138.450 399.144 138.450 307.616 L 138.450 216.089 152.170 216.089 L 165.889 216.089 165.983 277.712 C 166.085 344.848,165.985 340.777,167.705 348.104 C 173.399 372.358,192.654 391.629,216.856 397.298 C 223.922 398.953,223.170 398.918,253.948 399.035 L 282.509 399.144 282.509 307.635 L 282.509 216.126 306.642 216.005 C 333.410 215.872,332.591 215.926,340.836 213.731 C 363.395 207.724,381.433 189.556,387.442 166.790 C 389.487 159.042,389.476 159.244,389.591 127.897 L 389.695 99.483 336.121 99.483 L 282.548 99.483 282.421 80.221 C 282.283 59.270,282.274 59.135,280.460 51.928 C 274.515 28.309,255.528 9.702,231.587 4.035 C 224.695 2.403,223.745 2.362,193.638 2.362 L 165.904 2.362 165.904 50.923 L 165.904 99.483 152.202 99.483 L 138.500 99.483 138.373 81.107 C 138.235 61.106,138.074 58.899,136.249 51.893 C 129.283 25.165,106.644 5.580,79.347 2.667 C 77.584 2.479,65.966 2.362,49.161 2.362 L 21.841 2.362 21.917 171.587 M282.209 127.159 C 282.202 159.076,282.136 159.998,279.224 168.708 C 270.653 194.344,246.030 212.755,216.679 215.477 C 214.611 215.668,204.079 215.792,189.726 215.792 L 166.167 215.793 166.274 187.380 C 166.388 157.053,166.348 157.951,167.853 151.882 C 174.243 126.120,196.607 106.325,225.236 101.091 C 231.613 99.925,233.291 99.860,258.376 99.818 L 282.214 99.779 282.209 127.159 " stroke="none" fill="#000000" fill-rule="evenodd"></path>
+	</g>
+</svg>
```

Comment:
@conjurer-rich feel free to reuse it, this SVG is more optimized than the original one (which included a "white shape" to simulate the transparent part) 😇

Reply:
Hey @malparty thanks for taking a look at this. 

The build seems to failing with this reason:

file:///opt/build/repo/node_modules/vite/bin/vite.js:7
5:33:50 PM:     await import('source-map-support').then((r) => r.default.install())
5:33:50 PM:     ^^^^^
5:33:50 PM: SyntaxError: Unexpected reserved word
5:33:50 PM:     at Loader.moduleStrategy (internal/modules/esm/translators.js:81:18)
5:33:50 PM:     at async link

Reply:
@malparty it looks like this might be related to the node version on the Netlify agent: https://github.com/vitejs/vite/issues/9582. What version of node should it be using?

Is it worth adding a .node-version or .nvmrc file as per https://docs.netlify.com/configure-builds/manage-dependencies/?

Reply:
`.node-version` added. I'm currently using `16.14.2`. Let see if that works 👀

Reply:
Nope... did not solve the issue. 
Is there any settings in Netlify to configure the build command (`npm install` and then `npm run build`) — while it did not change, maybe Netlify was directly running the webpack command? 🤔